src/main/java/bio/terra/service/load/LoadDao.java

```diff
@@ -37,16 +57,41 @@ public Load lockLoad(String loadTag, String flightId) {
 
         if (load.isLocked()) {
             if (StringUtils.equals(flightId, load.getLockingFlightId())) {
+                System.out.println("SUCCESS: we have it locked");
```

Comment:
Could this and the other println's in this file use the logger instead of stdout? In some places below it looks like you used both the logger and stdout, was that just for debugging maybe?

Reply:
Yeah, that was debug stuff. I'll remove it.